api/RestfulService.php

```diff
@@ -53,7 +53,7 @@ public static function flush() {
 	 */
 	public static function set_default_curl_option($option, $value) {
 		Deprecation::notice('3.2', 'Use the "RestfulService.default_curl_options" config setting instead');
-		Config::inst()->update('RestfulService', 'default_curl_options', array($option => $value));
+		Config::inst()->update(get_called_class(), 'default_curl_options', array($option => $value));
```

Comment:
This (and other occurrences) can be replaced with `static::config()->update(...` too. Sorry to mess you around.